.idea/codeStyles/Project.xml

```diff
@@ -1,11 +1,27 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
-    <AndroidXmlCodeStyleSettings>
-      <option name="ARRANGEMENT_SETTINGS_MIGRATED_TO_191" value="true" />
-    </AndroidXmlCodeStyleSettings>
     <JetCodeStyleSettings>
+      <option name="PACKAGES_TO_USE_STAR_IMPORTS">
```

Comment:
These are system-specific changes. These changes should not be pushed in the PR.

Reply:
This has not been done yet.

Reply:
in this file, as i make changes then it will automatically move back to previous code, i don't know why it is happening.

Reply:
Removes these lines then commit the changes before these lines get added. After these lines get automatically added then don't commit those changes.